Makefile

```diff
@@ -179,13 +179,17 @@ test-ui-github-actions: reports setup-ui
 
 .PHONY: wave
 wave:
+	nvidia-smi && \
+	H2O_WAVE_APP_ACCESS_KEY_ID=dev \
+	H2O_WAVE_APP_ACCESS_KEY_SECRET=dev \
 	H2O_WAVE_MAX_REQUEST_SIZE=25MB \
 	H2O_WAVE_NO_LOG=true \
 	H2O_WAVE_PRIVATE_DIR="/download/@$(WORKDIR)/output/download" \
 	$(PIPENV) run wave run app
 
 .PHONY: llmstudio
 llmstudio:
+	nvidia-smi && \
```

Comment:
would it be enough to keep it only for `make llmstudio`?

Reply:
Yes, indeed not really needed for dev. Good point, will change it.

Reply:
Want to keep it for `llmstudio-conda`?

Reply:
yes, it seems that this isn't meant for dev, as it has the `--no-reload` flag?

Reply:
fair enough